### PR TITLE
fix: Removing payment fee on stripe

### LINKED ISSUE #10650: [CAL-2323] Remove stripe app fee
<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-2323](https://linear.app/calcom/issue/CAL-2323/remove-stripe-app-fee)</sub>

### PR DESCRIPTION
**Pull request by alannnc**
## What does this PR do?

- Removes from UI payment fees.
- Didn't moved envs so we can reuse them in the future if required.

Fixes #10650 

## Requirement/Documentation

- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

## Mandatory Tasks

- [X] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: packages/app-store/stripepayment/components/EventTypeAppCardInterface.tsx [hunk 1]
```diff
@@ -45,18 +45,7 @@ const EventTypeAppCard: EventTypeAppCardComponent = function EventTypeAppCard({
...
-          <div className="">
-            {t("payment_app_commission", {
-              paymentFeePercentage: 0.5,
-              fee: 0.1,
-              formatParams: { fee: { currency } },
-            })}
-          </div>
-        </>
-      }>
+      }}>
```
Comment (alannnc) [inline packages/app-store/stripepayment/components/EventTypeAppCardInterface.tsx:48]: I tried to find a link on stripe connect fees, but they are specific for each country.

File: packages/app-store/stripepayment/lib/PaymentService.ts [hunk 2]
```diff
@@ -102,7 +98,7 @@ export class PaymentService implements IAbstractPaymentService {
             stripe_publishable_key: this.credentials.stripe_publishable_key,
             stripeAccount: this.credentials.stripe_user_id,
           }) as unknown as Prisma.InputJsonValue,
-          fee: paymentFee,
+          fee: 0,
```
Comment (alannnc) [inline packages/app-store/stripepayment/lib/PaymentService.ts:101]: fee is required when creating payment.

**Comment by PeerRich** [comment]
when are we planning to reintroduce itß

**Comment by alannnc** [comment]
> when are we planning to reintroduce it?

We already talked about something, at the end of the month calculate total % of booking paid and then charge the user with that fee. 
Created RFC issue on linear: https://linear.app/calcom/issue/CAL-2356/rfc-calcom-booking-payment-fees
@PeerRich

**Review by zomars — APPROVED** [review]

**Pull request merged**